buildSrc/src/main/kotlin/convention.publishing.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import com.redmadrobot.build.dsl.credentialsExist
+import com.redmadrobot.build.dsl.developer
+import com.redmadrobot.build.dsl.githubPackages
+import com.redmadrobot.build.dsl.isReleaseVersion
+import com.redmadrobot.build.dsl.isRunningOnCi
+import com.redmadrobot.build.dsl.mit
+import com.redmadrobot.build.dsl.ossrh
+import com.redmadrobot.build.dsl.setGitHubProject
```

Comment:
Could you please update [.editorconfig](https://gist.github.com/osipxd/9a786496634a1ef0ea622c9805601490) in the project? I've added a directive to always force wildcard imports in scripts.